### DISCUSSION TITLE
Impossible to approve my Github Education.

### DISCUSSION THREAD

**Discussion by takapc** [community/community#53795] [GitHub Education]
I was sent "You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.” by github
Please help me

**Comment by santoshvandari** [comment]


**Reply by takapc** [reply]
Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
But I wrote my academic affiliation document plus a short bio.

**Reply by samus-aran** [reply]
@santoshvandari did you intent to link to the same discussion?

**Reply by gijoe250** [reply]
Also having the same problem where it is not recognizing my updated name and bio

**Comment by samus-aran** [comment]
Hi @takapc 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

Let us know how you get on 👍

**Comment by takapc** [comment]
Hi @samus-aran 
I changed my profile to the (probably) correct one, took a picture of it with my PC camera along with an English translation of my student ID and submitted my application, only to receive the same message.
